packages/transaction-controller/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Bump `remote-feature-flag-controleller` version up to 1.5.0
```

Comment:
Nit: Sorry my mistake no need for this bump. as this package has just been added with the below change
```suggestion